content/en/agent/logs/proxy.md

```diff
@@ -60,17 +63,236 @@ The parameter above can also be set with the following environment variable:
+    resolve_retries 3
+    timeout resolve 2s
+    timeout retry 1s
+    accepted_payload_size 8192
+    hold valid 10s
+    hold obsolete 60s
+    
+# This declares the endpoint where your Agents connects for
+# sending Logs (e.g the value of "logs.config.logs_dd_url")
+frontend logs_frontend
+    bind *:10514
+    mode tcp
+    option tcplog
+    default_backend datadog-logs
+    
+# This is the Datadog server. In effect any TCP request coming
+# to the forwarder frontends defined above are proxied to
+# Datadog's public endpoints.
+backend datadog-logs
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server datadog agent-intake.logs.datadoghq.com:10516 ssl verify required ca-file /etc/ssl/certs/ca-certificates.crt check port 10516
+```
+
+**Note**: Download the certificate with the following command:
+        * `sudo apt-get install ca-certificates` (Debian, Ubuntu)
+        * `yum install ca-certificates` (CentOS, Redhat)
+If successful, the file will be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
+
+Once the HAProxy configuration is in place, you can reload it or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.com` fails over to another IP.
+
+
+{{< /site-region >}}
+{{< site-region region="eu" >}}
+
+```conf
+# Basic configuration
+global
+    log 127.0.0.1 local0
+    maxconn 4096
+    stats socket /tmp/haproxy
+# Some sane defaults
+defaults
+    log     global
+    option  dontlognull
+    retries 3
+    option  redispatch
+    timeout client 5s
+    timeout server 5s
+    timeout connect 5s
+
+# This declares a view into HAProxy statistics, on port 3833
+# You do not need credentials to view this page and you can
+# turn it off once you are done with setup.
+listen stats
+    bind *:3833
+    mode http
+    stats enable
+    stats uri /
+
+# This section is to reload DNS Records
+# Replace <DNS_SERVER_IP> and <DNS_SECONDARY_SERVER_IP> with your DNS Server IP addresses.
+# For HAProxy 1.8 and newer
+resolvers my-dns
+    nameserver dns1 <DNS_SERVER_IP>:53
+    nameserver dns2 <DNS_SECONDARY_SERVER_IP>:53
+    resolve_retries 3
+    timeout resolve 2s
+    timeout retry 1s
+    accepted_payload_size 8192
+    hold valid 10s
+    hold obsolete 60s
+    
+# This declares the endpoint where your Agents connects for
+# sending Logs (e.g the value of "logs.config.logs_dd_url")
+frontend logs_frontend
+    bind *:10514
+    mode tcp
+    default_backend datadog-logs
+    
+# This is the Datadog server. In effect any TCP request coming
+# to the forwarder frontends defined above are proxied to
+# Datadog's public endpoints.
+backend datadog-logs
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server datadog agent-intake.logs.datadoghq.eu:443 ssl verify required ca-file /etc/ssl/certs/ca-bundle.crt check port 443
```

Comment:
This is the EU endpoint, how do we make sure that for US we have the right endpoint?

Reply:
See in the nginx example:

```
{{< /site-region >}}
{{< site-region region="eu" >}}
```

Reply:
Yep, correct example! It's `{{< site-region region="us" >}}` for US content, `{{< site-region region="eu" >}}` for EU content.

Reply:
sorry I don't get understand the comment. is there any problem with the current version? 
 US example is from line 82 to 149 and EU example is from 150 to 216  for HAProxy. Should I change something?

Reply:
Seems that both EU and US version points to the `.eu` endpoint currently.

So we would need to include the `site-region` header to make sure we display the right endpoint based on the region.

Reply:
If you format it like:

{{< site-region region="us" >}}
US content goes here
{{< /site-region >}}

{{< site-region region="us" >}}
EU content goes here
{{< /site-region >}}

It will switch the content based on region when someone uses the region selector in the top right of the docs page.

Reply:
double checked on slack with sarina, we are good :D